docs/scripts_md/MRI.md

```diff
@@ -0,0 +1,367 @@
+`&in_range()`
+
+RETURNS: scalar range SQL string
+
+### register\_db($file\_ref)
+
+Registers the NeuroDB::File object referenced by `$file_ref` into the database.
+
+INPUT: file hashref
+
+RETURNS: 0 if the file is already registered, the new FileID otherwise
+
+### mapDicomParameters($file\_ref)
+
+Maps DICOM parameters to more meaningful names in the NeuroDB::File object
+referenced by `$file_ref`.
+
+INPUT: file hashref
+
+### findScannerID($manufacturer, $model, $serialNumber, ...)
+
+Finds the scannerID for the scanner as defined by `$manufacturer`, `$model`,
+`$serialNumber`, `$softwareVersion`, using the database attached to the DBI
+database handle reference `$dbhr`. If no scannerID exists, one will be
+created.
+
+INPUTS:
+  - $manufacturer   : scanner's manufacturer
+  - $model          : scanner's model
+  - $serialNumber   : scanner's serial number
+  - $softwareVersion: scanner's software version
+  - $centerID       : scanner's center ID
+  - $dbhr           : database handle reference
+  - $register\_new   : if set, will call the function `&registerScanner`
+
+RETURNS: (int) scanner ID
+
+### registerScanner($manufacturer, $model, $serialNumber, ...)
+
+Registers the scanner as defined by `$manufacturer`, `$model`,
+`$serialNumber`, `$softwareVersion`, into the database attached to the DBI
+database handle reference `$dbhr`.
+
+INPUTS:
+  - $manufacturer   : scanner's manufacturer
+  - $model          : scanner's model
+  - $serialNumber   : scanner's serial number
+  - $softwareVersion: scanner's software version
+  - $centerID       : scanner's center ID
+  - $dbhr           : database handle reference
+
+RETURNS: (int) scanner ID
+
+### createNewCandID($dbhr)
+
+Creates a new CandID.
+
+INPUT: database handle reference
+
+RETURNS: (int) CandID
+
+### getPSC($patientName, $dbhr)
+
+Looks for the site alias in whatever field (usually patient\_name or
+patient\_id) is provided and return the MRI alias and CenterID.
+
+INPUTS: patient name, database handle reference
+
+RETURNS: a two element array:
+  - first is the MRI alias of the PSC or "UNKN"
+  - second is the centerID or 0
+
+### compute\_hash($file\_ref)
+
+Semi-intelligently generates a hash (MD5 digest) for the NeuroDB::File object
+referenced by `$file_ref`.
+
+INPUT: file hashref
+
+RETURNS: the generated MD5 hash
+
+### is\_unique\_hash($file\_ref)
+
+Determines if the file is unique using the hash (MD5 digest) from the
+NeuroDB::File object referenced by `$file_ref`.
+
+INPUT: file hashref
+
+RETURNS: 1 if the file is unique (or if hashes are not being tracked) or 0
+otherwise.
+
+### make\_pics($file\_ref, $data\_dir, $dest\_dir, $horizontalPics)
+
+Generates check pics for the Imaging Browser module for the `NeuroDB::File`
+object referenced by `$file_ref`.
+
+INPUTS:
+  - $file\_ref      : file hash ref
+  - $data\_dir      : data directory (C/data/$PROJECT/data> typically)
+  - $dest\_dir      : destination directory (`/data/$PROJECT/data` typically)
```

Comment:
@cmadjar 
why not specify that it is /data/..../pics? isn't that what our install does anyway?

Reply:
will do